.github/workflows/test.yml

```diff
@@ -51,15 +42,48 @@ jobs:
           mkdir qemu-data
           touch qemu-data/hdd.qcow
           ./qemu-run.sh nographic os &
-      - name: Create directory for logs
-        shell: bash
-        id: log_dirs
-        run: |
-          timestamp=$(date -u +%Y-%m-%dT%H:%M:%S%Z)
-          directory="/tmp/dts-test-ci-${timestamp}"
-          mkdir $directory
-          echo "directory=$directory" >> "$GITHUB_OUTPUT"
-      - name: Run tests
```

Comment:
Why did you remove this step? You should probably create folder per each robot run e.g.:

```
mkdir -p "/tmp/dst-test-ci-${{ github.run_id }}/${{ matrix.include.test }}"
```

so we can temporarily store logs in case of errors (otherwise cleanup job would remove logs)

Reply:
Added the step here: 2f8993cab0d0c4e27cf009abaaf0e2eb52333868

Reply:
```yaml
      - name: Create directory for logs
        shell: bash
        id: create_log_dirs
        run: |
          export LOG_DIR="/tmp/dst-test-ci-${{ github.run_id }}/${{ matrix.include.test }}"
          mkdir -p $LOG_DIR
          echo "LOG_DIR=$LOG_DIR" >> $GITHUB_ENV
      - name: Run ${{ matrix.include.test }}
        shell: bash
        env:
          LOG_DIR: ${{ steps.log_dirs.outputs.directory }}
```

In `Run` step you are trying to access `${{ steps.log_dirs.outputs.directory }}` but previous step has id `create_log_dirs` and outputted variable name is `LOG_DIR`

Reply:
Didn't check rest of steps, check and fix in all places if needed

Reply:
Fixed all LOG_DIR occurences:  60cee9b8bd69cb88a9b635884facbb6ce0687b7a